operator-common/src/test/java/io/strimzi/operator/common/featuregates/FeatureGatesTest.java

```diff
@@ -52,26 +52,30 @@ public void testAllFeatureGates() {
     }
 
     @ParallelTest
+    @Disabled("Once we add new FeatureGate we should enable this test test.")
     public void testFeatureGatesParsing() {
-        assertThat(new FeatureGates("+ContinueReconciliationOnManualRollingUpdateFailure").continueOnManualRUFailureEnabled(), is(true));
-        assertThat(new FeatureGates("-ContinueReconciliationOnManualRollingUpdateFailure").continueOnManualRUFailureEnabled(), is(false));
-        assertThat(new FeatureGates("  -ContinueReconciliationOnManualRollingUpdateFailure    ").continueOnManualRUFailureEnabled(), is(false));
+        // TODO: Once we have new FeatureGate, we should implement the checks as shown below
```

Comment:
Do we need to comment out the code if the test is disabled?

Reply:
I just wanted to disable the test so it's skipped and not executed at all (even if there is nothing actually).

Reply:
We can enable the test but keep the code commented -> there is nothing like `continueOnManualRUFailureEnabled()` method that would be used here.